.gitignore

```diff
@@ -1,68 +1,6 @@
-# # euler/__pycache__/Euler.cpython-37.pyc
-# # util_funcs/test/__pycache__/test_time_functions.cpython-37-pytest-5.0.1.pyc
-# # util_funcs/__pycache__/time_functions.cpython-37.pyc
-# # util_funcs/__pycache__/sun_utils.cpython-37.pyc
-# # util_funcs/__pycache__/frame_conversions.cpython-37.pyc
-# # util_funcs/__pycache__/__init__.cpython-37.pyc
-# # __pycache__/test_TRIAD.cpython-37-pytest-5.0.1.pyc
-# # TRIAD/__pycache__/deterministic_ad.cpython-37.pyc
-# # TRIAD/__pycache__/__init__.cpython-37.pyc
-
-# .cache/
-# .idea/
-# .vs/
-# cmake-build-debug/
-# documentation.wiki/
-
-# # orbit_propagation/orbit_prop_py/test/__pycache__/
-# # orbit_propagation/test/__pycache__/
-# # util_funcs/.cache/
-# # util_funcs/test/__pycache__/
-
-# # MEKF/MEKF_py/__pycache__/
-# # TRIAD/py_funcs/__pycache__/
-# # util_funcs/py_funcs/__pycache__/
-
-# _deps/pybind11-build/CMakeFiles/CMakeDirectoryInformation.cmake
-
-# _deps/pybind11-build/Makefile
-# _deps/
-# # CMakeFiles/
-
-# _deps/pybind11-subbuild/CMakeCache.txt
-
-# # util_funcs/__pycache__/
-
-# # TRIAD/test/__pycache__/
-
-# Makefile
-
-# # cmake_install.cmake
-
-# # __pycache__/conftest.cpython-36-PYTEST.pyc
-
-# CMakeCache.txt
-
-# # euler/__pycache__/__init__.cpython-36.pyc
-
-# # euler/__pycache__/Euler.cpython-36.pyc
-
-# # orbit_propagation/__pycache__/__init__.cpython-36.pyc
-
-# # orbit_propagation/orbit_prop_py/__pycache__/orbit.cpython-36.pyc
-
-# # detumble/__pycache__/__init__.cpython-36.pyc
-
-# # detumble/py_funcs/__pycache__/__init__.cpython-36.pyc
-
-# # detumble/py_funcs/__pycache__/detumble_algorithms.cpython-36.pyc
-
-# # detumble_cpp.cpython-36m-x86_64-linux-gnu.so
-
 /CMakeFiles/
 __pycache__
 *.pyc
-*.so
 *.cmake
 /_deps/
 _deps/pybind11-build/CMakeFiles/progress.marks
```

Comment:
Do we want to edit the .ignore file this much?

Reply:
I was just removing all of the commented out stuff and adding back in the created python modules so that the workflow can run properly. Was mainly just cleaning up the gitignore file.

Reply:
Ah gotcha, that works for me.